content/news/2022_01_30_GRASS_GIS_8_0_0_released.md

```diff
@@ -0,0 +1,179 @@
+embedded semantic labels. This allows to apply the signature file from
+one imagery scene (group) to any number of other scenes as long as they
+consist of the same bands (as identified by their semantic labels).
+This makes it so **much easier to classify different imagery groups 
+with identical semantic labels**.
+
+The new module 
+[i.band.library](https://grass.osgeo.org/grass80/manuals/i.band.library.html) 
+stores and prints available semantic labels used for multispectral 
+data. So far only system-defined band references are supported, 
+i.e., Landsat 5, Landsat 7, Landsat 8 and Sentinel 2.
+
+#### Temporal GIS modules
+
+With the addition of semantic labels, the temporal database was modified to
+version 3 and hence, to be able to read and process GRASS 7.x space-time
+datasets, users will be prompted to run the new 
+[t.upgrade](https://grass.osgeo.org/grass80/manuals/t.upgrade.html) module.
+If users want to read space-time datasets newly created in GRASS 8 back
+in GRASS 7.x, they can run
+[t.downgrade](https://grass.osgeo.org/grass78/manuals/t.downgrade.html).
+
+#### General modules
+
+[g.extension](https://grass.osgeo.org/grass80/manuals/g.extension.html),
+the tool allowing to install extensions, received extensive changes
+to adapt to the new [addons repository](https://github.com/OSGeo/grass-addons) 
+structure with branches pointing to different GRASS major versions.
+This change introduced the possibility to **install pull request branches**
+making it so much faster to test new changes.
+
+#### Scripting and code quality
+
+Major efforts have been made to port all Python scripts to Python 3.6.
+Also, there's a new Python subpackage for working with locations and mapsets.
+All Python code has been formatted with `black` and checked against `flake8`.
+Most warnings in C code were removed and are now checked and, all code 
+is tested with CodeQL.
+
+#### PROJ and GDAL support
+
+PROJ 4 to PROJ 7 compatibility has been implemented. WKT2 can be 
+used for CRS definitions. High-accuracy coordinate transformations supported. 
+GDAL 3 compatibility improved with speed-ups.
+
+#### Full list of changes and new features
+
+The overview of new features in the 8.0 release series is available at
+[new features in GRASS GIS 8.0](https://trac.osgeo.org/grass/wiki/Grass7/NewFeatures80).
+See also our detailed announcement with the full list of changes and
+bugs fixed at
+[https://trac.osgeo.org/grass/wiki/Release/8.0.0-News](https://trac.osgeo.org/grass/wiki/Release/8.0.0-News).
+
+## Download and test!
+
+#### Binaries/Installers download:
+
+- winGRASS 8.0.0/standalone:
+  [64bit standalone installer](/grass80/binary/mswindows/native/WinGRASS-8.0.0-1-Setup.exe)
+- winGRASS 8.0.0/OSGeo4W:
+  [64bit OSGeo4W installer](http://download.osgeo.org/osgeo4w/v2/osgeo4w-setup.exe)
+- Mac
+    - [Official Application Bundle](http://grassmac.wikidot.com/downloads)
+    - ~~ [MacPorts](https://ports.macports.org/port/grass7/) ~~
```

Comment:
```suggestion
    -  [MacPorts](https://ports.macports.org/port/grass/)
```

I plan to suggest to revive the un-versioned `grass` port for GRASS v8 (it has been a, now obsolete, remnant of the v6 era).